c.pod_design.md

```diff
@@ -236,6 +236,9 @@ kubectl get po # get all the pods
 # OR you can find pods directly by:
 kubectl get po -l run=nginx # if you created deployment by 'run' command
 kubectl get po -l app=nginx # if you created deployment by 'create' command
+# OR you can describe the ReplicaSet to find the pods linked with it first and then get its yaml
+kubectl describe rs nginx-7bf7478b77 | grep -i pod
```

Comment:
To use that, one would first have to `kubectl get rs` and then use the ReplicaSet name.
Using  `nginx-7bf7478b77` in the example might be a bit misguiding for the reader, since if they to copy/paste it, it will result in an error (because their ReplicaSet will have a different name).